data-exporter/pom.xml

```diff
@@ -0,0 +1,211 @@
+		<reactome-graph-core.version>1.1.17</reactome-graph-core.version>
+		<neo4j.version>3.4.9</neo4j.version>
+		<neo4j-java-driver.version>1.7.2</neo4j-java-driver.version>
+		<junit.version>5.4.0-M1</junit.version>
+		<hamcrest.version>2.1</hamcrest.version>
+		<mockito.version>2.23.4</mockito.version>
+	</properties>
+
+	<dependencies>
+		<!-- Main dependencies -->
+		<dependency>
+			<groupId>org.reactome.release</groupId>
+			<artifactId>release-common-lib</artifactId>
+			<version>${reactome-release-common-lib.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.reactome.base</groupId>
+			<artifactId>reactome-base</artifactId>
+			<version>RELEASE</version>
+		</dependency>
+		<dependency>
+			<groupId>org.reactome.server.graph</groupId>
+			<artifactId>graph-core</artifactId>
+			<version>${reactome-graph-core.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.neo4j.driver</groupId>
+			<artifactId>neo4j-java-driver</artifactId>
+			<version>${neo4j-java-driver.version}</version>
+		</dependency>
+
+		<!-- Test dependencies -->
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-api</artifactId>
+			<version>${junit.version}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-engine</artifactId>
+			<version>${junit.version}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.hamcrest</groupId>
+			<artifactId>hamcrest</artifactId>
+			<version>${hamcrest.version}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-core</artifactId>
+			<version>${mockito.version}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.neo4j.test</groupId>
+			<artifactId>neo4j-harness</artifactId>
+			<version>${neo4j.version}</version>
+			<scope>test</scope>
+		</dependency>
+
+		<!-- Compilation dependencies -->
+		<dependency>
+			<groupId>org.jetbrains</groupId>
+			<artifactId>annotations</artifactId>
+			<version>16.0.2</version>
+			<scope>compile</scope>
+		</dependency>
+	</dependencies>
+
+	<build>
+		<plugins>
+			<plugin>
+				<artifactId>maven-assembly-plugin</artifactId>
+				<configuration>
+					<archive>
+						<manifest>
+							<mainClass>org.reactome.release.Main</mainClass>
+						</manifest>
+					</archive>
+					<descriptorRefs>
+						<descriptorRef>jar-with-dependencies</descriptorRef>
+					</descriptorRefs>
+				</configuration>
+				<executions>
+					<execution>
+						<id>make-assembly</id> <!-- this is used for inheritance merges -->
+						<phase>package</phase> <!-- bind to the packaging phase -->
+						<goals>
+							<goal>single</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-help-plugin</artifactId>
```

Comment:
What is this plugin for?

Reply:
Eclipse complains:
> Plugin execution not covered by lifecycle configuration: org.apache.maven.plugins:maven-help-plugin:3.1.0:active-profiles (execution: show-profiles, phase: compile)

And if I comment this plugin out, the project builds just fine. So, what does it do?

Reply:
I think this was a carryover from another POM file I used as a template.  I'll remove it.